NumSet/NumSet/Writer/SetWriter.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Assingment4.Writer
+{
+    public class SetWriter : IDisposable
+    {
+        private bool _DisposedValue;
```

Comment:
You can set this to be false by default.
```suggestion
        private bool _DisposedValue = false;
```